erda.yml

```diff
@@ -518,6 +518,7 @@ services:
       UC_CLIENT_SECRET: "secret"
       ERDA_NAMESPACE: "default"
       ERDA_HELM_CHART_VERSION: "1.1.0"
+      CMP_ADDR: "cmp:9027"
```

Comment:
Add env in client side. Or globally.

Reply:
cmp register self to eventbox
add it to global envs